mirgecom/initializers.py

```diff
@@ -634,13 +635,11 @@ class MulticomponentLump:
     .. automethod:: exact_rhs
     """
 
-    def __init__(
-            self, *, dim=1, nspecies=0,
-            rho0=1.0, p0=1.0,
-            center=None, velocity=None,
-            spec_y0s=None, spec_amplitudes=None,
-            spec_centers=None, sigma=1.0
-    ):
+    def __init__(self, *, dim=1, nspecies=0,
+                 rho0=1.0, p0=1.0,
+                 center=None, velocity=None,
+                 spec_y0s=None, spec_amplitudes=None,
+                 spec_centers=None, sigma=1.0):
```

Comment:
(Personal preference, I know, but ugh -- I hate the parentheses-aligned arguments thing. Shifts everything over so far.)